_i18n/de/resources/user-guides/mine-to-pool.md

```diff
@@ -151,94 +127,70 @@ After these modifications, your configuration should look something like this:
 
-On some CPUs, XMRig tries to increase performance by disabling certain
-features like your CPU's instruction prefetcher. These operations
-require root/administrator, so try right clicking xmrig.exe and
-running it as administrator, or running `sudo ./xmrig` on other
-systems.
+Bei manchen CPUs versucht XMRig es, die Leistung durch Deaktivierung bestimmter Funktionen (etwa des Anweisungsprefetchers deines CPUs) zu verbessern. Diese Vorgänge benötigen Administratorrechte beziehungsweise den Gebrauch des Root-Kontos. Versuche, es mit einem Rechtsklick auf xmrig.exe als Administrator auszuführen oder es auf anderen Systemen durch den Befehl `sudo ./xmrig` zu starten.
 
-### Algo not known
+### Algo nicht bekannt
 
-Find the line in config.json that says `algo: null` and change it to
-`algo: "rx/0"`. By default, XMRig expects the pool to tell it which
-hashing algorithm to use.
+Suche in der "config.json"-Datei nach der Zeile, die `algo: null` enthält, und ändere dies zu `algo: "rx/0"`. Standardmäßig wartet XMRig nun darauf, dass der Pool mitteilt, welcher Hashing-Algorithmus verwendet werden soll.
 
-### Huge Pages 0%
+### Große Seiten 0%
 
-#### Allowing large pages on Windows
+#### Große Seiten auf Windows erlauben
 
-Taken from [the MSDN](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/enable-the-lock-pages-in-memory-option-windows?view=sql-server-ver15):
+Von [MDSN](https://docs.microsoft.com/de-de/sql/database-engine/configure-windows/enable-the-lock-pages-in-memory-option-windows?view=sql-server-ver15) übernommen:
 
->1. On the Start menu, click Run. In the Open box, type gpedit.msc.
-2. On the Local Group Policy Editor console, expand Computer Configuration, and then expand Windows Settings.
-3. Expand Security Settings, and then expand Local Policies.
-4. Select the User Rights Assignment folder.
-5. The policies will be displayed in the details pane.
-6. In the pane, double-click Lock pages in memory.
-7. In the Local Security Setting – Lock pages in memory dialog box, click Add User or Group.
-8. In the Select Users, Service Accounts, or Groups dialog box, add an account that you will run the miner on
-9. Reboot for change to take effect.
+>1. Klicken Sie im Menü "Start" auf "Ausführen". Geben Sie "gpedit.msc" im Feld "Öffnen" ein.
+2. Erweitern Sie in der Konsole "Editor für lokale Gruppenrichtlinien" die Option "Computerkonfiguration" und dann "Windows-Einstellungen".
+3. Erweitern Sie "Sicherheitseinstellungen" und dann "Lokale Richtlinien".
+4. Wählen Sie den Ordner "Zuweisen von Benutzerrechten" aus. 
+Die Richtlinien werden im Detailbereich angezeigt.
+5. Doppelklicken Sie im Detailbereich auf "Sperren von Seiten im Speicher".
+6. Klicken Sie im Dialogfeld "Lokale Sicherheitseinstellung" > "Sperren von Seiten im Speicher auf Benutzer oder Gruppe hinzufügen".
+7. Wählen Sie im Dialogfeld "Select Users, Service Accounts, or Groups" (Benutzer, Dienstkonten oder Gruppen auswählen) das SQL Server-Dienstkonto aus.
```

Comment:
```suggestion
7. Wählen Sie im Dialogfeld "Select Users, Service Accounts, or Groups" (Benutzer, Dienstkonten oder Gruppen auswählen) das Benutzerkonto aus, unter welchem der Miner ausgeführt werden soll.
```

Reply:
All this is taken from the [official German translation of the MDSN](https://docs.microsoft.com/de-de/sql/database-engine/configure-windows/enable-the-lock-pages-in-memory-option-windows?view=sql-server-ver15), like the English version. Otherwise there wouldn't be this annoying, stiff "Sie-Form". :)
I could add your suggestions in square brackets so it would be compliant with citation rules..?

Reply:
@stmax82, sorry to bother you (hopefully one last time): I just opened another pull request with the changes of this part, maybe you could have a look. :)

Reply:
#1396? I'll take a look!

Reply:
The adjusted translation looks good to me. I didn't know this was a copy from MSDN. That explains how the "SQL Server" got in there :)